pkg/transport/managed_transport.go

```diff
@@ -66,16 +56,8 @@ type ManagedTransport struct {
 	LogEntry   *LogEntry
 	logUpdates uint32
 
-	dc  DiscoveryClient
-	ls  LogStore
-	ebc *appevent.Broadcaster
```

Comment:
I checked and they were not used anywhere else except for some tcp events. I moved those events to transport manager itself to avoid passing those around, but perhaps they should be kept here.